index.js

```diff
@@ -27,3 +27,18 @@ function closeNav() {
 hamburgerButton.addEventListener('click', openNav);
 closeButton.addEventListener('click', closeNav);
 navItems.addEventListener('click', closeNav);
+
+function emailValidation(email, error, event) {
+  if (email !== email.toLowerCase()) {
+    event.preventDefault();
+    const errMessage = document.querySelector('.validation-message');
+    errMessage.innerText = error;
+  }
+}
+const form = document.getElementById('form');
+form.addEventListener('submit', (event) => {
+  const invalidEmail = 'Error, please consider to write all lower case letters.';
+  const email = form.elements.mail;
```

Comment:
![Capture1](https://user-images.githubusercontent.com/93039359/209291845-51eca001-5953-4aa0-a11d-4bd79fa4b7aa.PNG)
**[Optional]**

- [ ] It's great that you have implemented the validation 😊 but your message is not meaningful enough means you are not telling the user where to use all the lower case letters means in which text field. So you can say Please consider to write all lower case letters for email 👍